Yank/analyze.py

```diff
@@ -105,7 +105,35 @@ def get_standard_state_correction(self):
             self._computed_observables['standard_state_correction'] = ssc
         return self._computed_observables['standard_state_correction']
 
-    def analyze_phase(self, cutoff=0.05):
+    def analyze_phase(self, show_mixing=False, cutoff=0.05):
```

Comment:
I don't think the new `show_mixing` argument is used here.

Reply:
Yeah, I saw that in my testing, easy fix.